pom.xml

```diff
@@ -71,6 +71,15 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>version.properties</include>
```

Comment:
So I understand that you are using at the end a version.propertines.

That's fine but... just for curiosity, which technique is using @frbattid for implementing /version in Cygnus?